app/views/static_pages/home.html.erb

```diff
@@ -1,3 +1,19 @@
+<% if logged_in? %>
+  <div class="row">
+    <aside class="col-md-4">
+      <section class="user_info">
+        <%= render "shared/user_info" %>
+      </section>
+      <section class="micropost_form">
+        <%= render "shared/micropost_form", micropost: @micropost %>
+      </section>
+    </aside>
+    <div class="col-md-8">
+      <h3><%= t ".h3" %></h3>
+      <%= render "shared/feed", feed_items: @feed_items %>
+    </div>
+  </div>
+<% else %>
 <% provide :title, t(".title") %>
```

Comment:
nếu chưa  login thì vẫn cho người dungfxem tất user bài viiest của user khác thử xem

Reply:
cái này t chưa hiểu lắm, nếu chưa login mà đánh đường dẫn http://localhost:3000/vi/users/3
thì vẫn thấy được post mà

Reply:
cái này t chưa hiểu lắm, ý c là hiển thị nút users và cho xem cả khi chưa log in thế này à
![chap13 1](https://user-images.githubusercontent.com/19527303/35784505-cf8b7942-0a4a-11e8-8c78-3bda71d68e0b.png)
![Uploading Chap13.2.png…]()

Reply:
đúng rồi, còn indent ở đó nhé